views/register.ejs

```diff
@@ -0,0 +1,29 @@
+<body class="register_page">
+    <%- include ('./partials/head') -%>

```

Comment:
As mentioned earlier, `<head>` should be above body tag

Reply:
Ahh okay, did not see that, will change.

Reply:
pog